aggregator/apps/api/v1/docs/elections_endpoints.apibp

```diff
@@ -26,7 +26,7 @@ List or filter all election objects.
 
 + Parameters
     + coords: `52.290719,-1.935395` (string, optional) - Pass a lat/lon string as `?coords={lat},{lon}` to find elections covering the given coordinates. For lookup by postcode/address, use the [/postcode](#postcode-search-2) endpoint.
-    + current: `1` (string, optional) - Pass `?current=1` to return only elections whose polling date is either in the future or was in the last 20 days
+    + current: `1` (string, optional) - Pass `?current=1` to return only elections whose polling date is either in the next 90 days or was in the last 20 days
```

Comment:
As per my comment in https://github.com/DemocracyClub/EveryElection/pull/597, might it be worth expanding on the wording a bit here to explain why this is useful / interesting? And that it's basically a thing we make an editorial decision about?